contrib/terraform/openstack/README.md

```diff
@@ -85,6 +87,28 @@ number_of_k8s_nodes = "0"
 ```
 This will provision one VM as master using a floating ip, two additional masters using no floating ips (these will only have private ips inside your tenancy) and one VM as node, again without a floating ip.
 
+##### Alternative: etcd on separate machines
+
+If you want to provision master or node VMs that don't use floating ips and where **etcd is not in the same machines as masters**, write on a `my-terraform-vars.tfvars` file, for example:
```

Comment:
Can you fix the wording a bit?
Change: 
```
**etcd is not in the same machines as masters**, write on a `my-terraform-vars.tfvars` file, for example:`
```
To: 

```
**etcd is on separate nodes from Kubernetes masters**, you can create a `my-terraform-vars.tfvars` file. Here is an example example:
```

Reply:
Thanks for reviewing it! Will go through suggested changes.

Reply:
I have modified now the commit to reflect your suggestion.